docs/usage/configuration.md

```diff
@@ -52,6 +52,17 @@ import python_ta
 python_ta.check_all(..., load_default_config=False)
 ```
 
+## Allowing 'pylint:' Comments
+
+PythonTA allows you to choose whether you want to locally disable checks using 'pylint:' or not, i.e. it
+lets you choose whether or not you want to allow comments that begin with 'pylint:'.
```

Comment:
Good! Please add a sentence that the default value is `False`, and explain that this means PythonTA will not check files that contain such comments.